inventory.yml.sample

```diff
@@ -182,7 +182,7 @@ all:
 
     vm_hosts:
       hosts:
-        vm_host: # Required for using "KVM" nodes, ignored if not.
+        my_vm_host: # Required for using "KVM" nodes, ignored if not.
```

Comment:
I think if we added a number to the sample `vm_host` name, it would give indication to the user that more than one `vm_host` is possible. One could say it shouldn't be needed because the group is named `vm_hosts` (plural), and to that I'd say: `bastions`.

We could name it `vm_host1`, as we already use (`super1`, `super2`, ...) and (`worker1`, `worker2`, ...) in our sample inventory.